lib/block-supports/typography.php

```diff
@@ -260,6 +260,7 @@ function gutenberg_render_typography_support( $block_content, $block ) {
 				$block_content = $processor->get_updated_html();
 			}
 		}
+		return $block_content;
```

Comment:
The rest of the code returns $block_content anyway which on this case we want to ignore so we should return here.

Reply:
Let's make this early return clear for other devs and mitigate the risk that future changes break anything:

```suggestion
        // fitText supersedes any other typography features
		return $block_content;
```

* * *

Also, shouldn't the condition at the top be improved (regardless of this PR)? Currently it works on the assumption that the default will always be false (and thus if the attribute has a value it must be true), but that sounds just a little fragile to me.

```diff
-if ( ! empty( $block['attrs']['fitText'] ) && ! is_admin() ) {
+if ( ! empty( $block['attrs']['fitText'] ) && $block['attrs']['fitText'] && ! is_admin() ) {
```

Reply:
Feedback applied 👍

Reply:
I don't know where my mind was last week, but my advice about `! empty( … )` was totally idiotic. Your code was fine for the purpose, @jorgefilipecosta. Sorry about that. :)

> A variable is considered empty if it does not exist [or if its value equals false](https://www.php.net/manual/en/function.empty.php#:~:text=A%20variable%20is%20considered%20empty%20if%20it%20does%20not%20exist%20or%20if%20its%20value%20equals%20false.%20empty()%20does%20not%20generate%20a%20warning%20if%20the%20variable%20does%20not%20exist.). empty() does not generate a warning if the variable does not exist.